crates/algo_sdk/src/main.rs

```diff
@@ -0,0 +1,19 @@
+// Let's Build advanced sdk functionality here.
+// Typescript is a fail, we will have to hand-craft the bindings so just ignore JS/TS
```

Comment:
What do you mean by "Typescript is a fail"?

Reply:
The fail is for automagical binding interfaces in the native targets like the Python example. Mainly to remind my self that "Typescript is not like the others"

This is mainly for UniFFI, I could have misinterpreted the status. I saw the JS library mainly integrated via wasm-pack and only found `kotlin, swift, python, ruby` as options in UniFFI.

Reply:
Ah gotcha. Yeah worth mentioning that Mozilla does have a JS binding generator for Gecko: https://firefox-source-docs.mozilla.org/rust-components/uniffi.html but don't think this includes TypeScript. There's also Neon, but I generally think that it'd be better to ship one lib for node and web rather than having two seperate packages. The main question is how viable using WASM in an npm package intended for web. Top-level await in a ES module to load the WASM seems ideal, but don't have enough domain knowledge to know the full implications of that (i.e. CJS compat). There's also the obvious factor: WASM binary size.